src/extensions/spdlog/SpdlogToActs.h

```diff
@@ -11,31 +11,110 @@
 #include <JANA/JException.h>
 
 namespace eicrecon {
-    inline Acts::Logging::Level SpdlogToActsLevel(spdlog::level::level_enum input) {
-
-        // Convert the source string to lower case
-        switch (input) {
-            case spdlog::level::trace:
-                return Acts::Logging::VERBOSE;
-            case spdlog::level::debug:
-                return Acts::Logging::DEBUG;
-            case spdlog::level::info:
-                return Acts::Logging::INFO;
-            case spdlog::level::warn:
-                return Acts::Logging::WARNING;
-            case spdlog::level::err:
-                return Acts::Logging::ERROR;
-            case spdlog::level::critical:
-                return Acts::Logging::FATAL;
-            case spdlog::level::off:
-                return Acts::Logging::FATAL;
-            case spdlog::level::n_levels:
-                [[fallthrough]];
-            default:
-                break;
-        }
-
-        auto err_msg = fmt::format("SpdlogToActsLevel don't know this log level: '{}'", input);
-        throw JException(err_msg);
+
```

Comment:
This should probably be a boost bimap with much simpler to and from functions.

Ref https://www.boost.org/doc/libs/1_82_0/libs/bimap/doc/html/boost_bimap/examples/simple_bimap.html